front/app/modules/commercial/content_builder/services/ContentBuilder.ts

```diff
@@ -0,0 +1,48 @@
+import { API_PATH } from 'containers/App/constants';
+import streams from 'utils/streams';
+
+export interface IBuilderLayoutData {
+  type: 'project';
+  id: string;
+  attributes: {
+    craftjs_jsonmultiloc: JSON;
```

Comment:
This is `Multiloc` type, no?

Reply:
@IvaKop I was a bit confused by the specification here: "craftjs_jsonmultiloc: A multiloc attribute, but instead of having string values, it has JSON values.". But perhaps that is just referring to the backend then? I will change it to a Multiloc type.

Reply:
Ah, you are right - so you need to type it the same way as Multiloc but the values are an object.

```
type JsonMultiloc = {
  [key in Locale]?: Record<string, unknown>;
};

Reply:
@IvaKop Just to confirm - this new JsonMultiloc type is added to the ContentBuilder.ts file, correct? Or should I declare this somewhere else?

Reply:
Yep, in the same file is fine. Files for services are usually named with a small letter though - maybe change that? It's a way to differentiate between components and the rest.